examples/01_plotting/plot_3d_map_to_surface_projection.py

```diff
@@ -0,0 +1,49 @@
+"""
+Making a surface plot of a 3D statistical map
+=============================================
+
+project a 3D statistical map onto a cortical mesh and display a surface plot.
+
+"""
+
+from nilearn import datasets, surface, plotting
```

Comment:
We should break out these imports in the cells where they are used. It's easier for beginners.